ci-operator/step-registry/openshift-extended/test/clusterinfrastructure/openshift-extended-test-clusterinfrastructure-ref.yaml

```diff
@@ -0,0 +1,32 @@
+ref:
+  as: openshift-extended-test-clusterinfrastructure
+  from: cluster-api-actuator-pkg-test
```

Comment:
@sunzhaohua2 frankly it has nothing with tests-private repo and its image.
so, I do not suggest to put this step here. you'd better to put it your own module.

and also @jianzhangbjz @Xia-Zhao-rh and me are not owner of this step.
so, please do not put us into reviewer and approver.

Reply:
@sunzhaohua2 please add our team members as owner

Reply:
Thanks @kuiwang02 @shellyyang1989 do you think move this step to` step-registry/openshift/e2e/test/clusterinfra-qe` is ok? cc @liangxia